.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please keep this file in .gitignore (as all IDE and editors configuration files)

Reply:
:) Got this when syncing layout branch with this. I dont use this feature, redas says he want to use same configs at home and at work.